scripts/iso-menu-session

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+dmesg -c
```

Comment:
After poking around for some related stuff, we might want to run "dmesg -n 4" here too (which I think sets the log level to the equivalent of putting "quiet" on the kernel command line)

Reply:
This turned out to be a hard problem.  I have implemented chvt games to work around this.  The short answer is that stdout prints before the agetty risk bleeding into the terminal.
I have hopes that the above works for the Subiquity version of the problem.